src/features/pagination/js/pagination.js

```diff
@@ -146,12 +176,14 @@
             var visibleRows = renderableRows.filter(function (row) { return row.visible; });
             grid.options.totalItems = visibleRows.length;
 
-            var firstRow = (currentPage - 1) * pageSize;
+            var firstRow = publicApi.methods.pagination.getFirstRowIndex();
+            var lastRow  = publicApi.methods.pagination.getLastRowIndex();
```

Comment:
that is what I meant. But I suppose you are right and we are not using this anywhere else in this project right now. It is my personal preference. No need to change it, in that case.